### PR TITLE
Add auto-prefixer

### LINKED ISSUE #3290: Broken icon on official gov site banner
## Expected Behavior
The banner should display a down chevron 
image

## Actual Behavior
Displaying a horizontal bar
image

I am guessing it has to do with this `mask` property
image

### PR DESCRIPTION
**Pull request by igorkorenfeld**
Adds in the autoprefixer gem, addressing #3317 . The settings are copied over from USWDS. Should close #3290 as well.

:sunglasses: &nbsp; [Preview](https://federalist-d703a102-c627-4805-8934-78f5bf4c97da.app.cloud.gov/preview/igorkorenfeld/18f.gsa.gov/ik-autoprefix/)

/cc @apburnes

### CODE REVIEW

File: admin/config.yml [hunk 1]
```diff
@@ -12,6 +12,13 @@ public_folder: /assets/blog
 publish_mode: editorial_workflow
 logo_url: https://18f.gsa.gov/assets/img/logos/18f-logo.svg
 
+autoprefixer:
```
Comment (apburnes) [inline admin/config.yml:15]: I think this config needs to be in the `_config.yml`. This config is for the Netlify CMS and not Jekyll.

**Review by apburnes — COMMENTED** [review]
Thanks Igor! Looks pretty good. I think we just need to move the autoprefixer settings into the Jekyll config.

File: admin/config.yml (hunk 1 above)
Comment (igorkorenfeld) [reply admin/config.yml]: Oh my mistake, thanks for catching that.

**Comment by igorkorenfeld** [comment]
@apburnes pushed up the updates. Just wanted to double check my understanding, the new gem does need to be in the general `Gemfile` in addition to the `GemfileFederalist` right?

File: admin/config.yml (hunk 1 above)
Comment (apburnes) [reply admin/config.yml]: Awesome!

**Comment by apburnes** [comment]
That's right @igorkorenfeld. We need the `GemfileFederalist` because one of the test dependencies cannot be installed in the Federalist build environment.  I think there is a more elegant way of requiring dependencies based on build type within the `Gemfile` but I have not had time to figure it out.

**Review by apburnes — APPROVED** [review]

**Comment by Dahianna** [comment]
Thanks @igorkorenfeld !

**Pull request merged**